src/App.tsx

```diff
@@ -1,18 +1,23 @@
-import { Fragment } from 'react';
+import React, { Fragment, Suspense } from 'react';
 import GlobalStyle, { theme } from './styles/global';
 import { ChakraProvider } from "@chakra-ui/react";
 import { Layout } from './containers/Layout/Layout';
 import { Redirect, Route, Switch } from 'react-router';
-import { MainPage } from './components/MainPage/Mainpage';
-import { GamePage } from './components/GamePage/GamePage';
+import { MainPage } from './components/MainPage/MainPage';
```

Comment:
There is something pretty nice about TypeScript called Barrel File, basically, in the root of /components, you can create a file index.ts and inside it you can do like:

export { MainPage } from "./MainPage"
export { GamePage } from "./GamePage"

Them in other screens, etc, if you want to call both you can simple do:

import { MainPage, GamePage } from "./components"

Reply:
I see, thanks... It seems like what I was doing in redux when exporting only needed actionCreators...
`export {
  logout,
  auth,
  checkAuthState,
  checkAuthErrorState
} from './auth';

export {
  newItem,
  setInitialState,
  fetchData,
  deleteItem,
  setItemEditMode,
  updateItem
} from './items';

export {
  newCollection,
  deleteCollection,
  addItemsToCollection,
  removeItemFromCollection,
  renameCollection
} from './collections';`